lib/util.js

```diff
@@ -67,6 +73,11 @@ module.exports.kickoff = function kickoff() {
           console.log(results);
           /* eslint-enable */
         }
+        done();
       }.bind(this));
+    }.bind(this);
   }.bind(this));
+
+  maxConcurrent = (baseProfile && baseProfile.maxConcurrent) || Infinity;
```

Comment:
Well, it was recommended to me by the team that manages our selenium-grid to hover around 75. I'm not opposed to capping it at something, but unbounded by default seems reasonable.